.github/workflows/ci.yml

```diff
@@ -71,6 +71,10 @@ jobs:
         with:
           files: ./results/UnitTest/junit.xml
 
+      #########################################################
+      # This step was disabled temprorly due the required write
```

Comment:
Spelling: 
`temprorly` -> `temporarily`
`... due the required ...` -> `... due to the required ...`
`... of the PR's for ...` -> `... of the PRs ..`
`... for the external contributors.`-> `... for external contributors.`

Reply:
Done!